yumljira/apps/projects/migrations/0007_auto_20191002_0941.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.5 on 2019-10-02 09:41
+
+from django.db import migrations, models
+import django.db.models.deletion
```

Comment:
We have to squash all of those migrations to initial later after this pull request I think